controller/rest/rest.go

```diff
@@ -1759,6 +1759,12 @@ func StartRESTServer() {
 	r.DELETE("/v1/api_key/:name", handlerApikeyDelete)
 	r.GET("/v1/selfapikey", handlerSelfApikeyShow) // Skip API document
 
+	// remote export repository
```

Comment:
How does UI list all the configured remote repository entries?
Why not follow the similar path as webhook APIs like
r.POST("/v1/system/config/webhook", handlerSystemWebhookCreate)
r.PATCH("/v1/system/config/webhook/:name", handlerSystemWebhookConfig)
r.DELETE("/v1/system/config/webhook/:name", handlerSystemWebhookDelete)
?

Reply:
The remote repositories are now part of the `system/config` object and can be retrieved as such.